Src/MainLib/InterceptFrame.cpp

```diff
@@ -903,13 +908,45 @@ bool InterceptFrame::IsBufferReadable(GLenum bufType, uint bufferCount) const
   return false;
 }
 
+///////////////////////////////////////////////////////////////////////////////
+//
+void InterceptFrame::GetBufferViewSize(GLint viewSize[4]) const
+{
+#ifdef GLI_BUILD_WINDOWS
+  if (this->fullscreen)
+  {
+      GLint fboId = 0;
+      GLV.glGetIntegerv(GL_READ_FRAMEBUFFER_BINDING, &fboId);
+      if (fboId == 0)
+      {
+          HDC hdc = GLW.wglGetCurrentDC();
+          HWND hwnd = WindowFromDC(hdc);
+          if (hwnd != nullptr)
+          {
+              RECT clientRect{};
+              if (GetClientRect(hwnd, &clientRect))
+              {
+                  viewSize[0] = 0;
+                  viewSize[1] = 0;
+                  viewSize[2] = clientRect.right - clientRect.left;
+                  viewSize[3] = clientRect.bottom - clientRect.top;
+                  return;
+              }
+          }
+      }
+  }
+#endif
+
```

Comment:
If you wanted to add support for FBOs with fullscreen you could add a call to GetRenderbufferParameteriv with GL_RENDERBUFFER_WIDTH (optional)

Reply:
Hm, isn't it more complicated?

Based on [this](https://www.khronos.org/opengl/wiki/Framebuffer_Object#Attachment_Completeness),

> Notice that there is no restriction based on size. The effective size of the FBO is the intersection of all of the sizes of the bound images (ie: the smallest in each dimension). 

we would have to iterate over all attachments, check their types (we don't now whether it's a texture or a renderbuffer), query their sizes and then take the minimum.

Reply:
I will fiddle with this when I have more free time and get back to you later :) For now I just needed this functionality for default framebuffers.

Reply:
You don't have to do this, it was just a suggestion.

Reply:
I added support for FBO!